### DISCUSSION TITLE
LOADING_DOCUMENT_FAILED

### DISCUSSION THREAD

**Discussion by liuhaichaogithub** [eclipse-edc/Connector#3673] [Q&A]
use [Transfer sample 02](https://eclipse-edc.github.io/docs/#/documentation/docs/samples/transfer/transfer-02-consumer-pull/README)

Negotiate  contract faild

![image](https://github.com/eclipse-edc/Connector/assets/40063899/9d6cd508-00d5-4da0-90aa-ab09d62e3847)
![Uploading image.png…]()

**Comment by liuhaichaogithub** [comment]
![image](https://github.com/eclipse-edc/Connector/assets/40063899/32afc669-e9ce-486e-a258-86c07994bd26)

**Comment by liuhaichaogithub** [comment]
![image](https://github.com/eclipse-edc/Connector/assets/40063899/18fd3b97-e7b4-45c0-a468-f4cc9b21221a)

**Comment by ndr-brt** [comment]
Was there anything on the connector console output? Operating system used?
